sklearn/ensemble/bagging.py

```diff
@@ -33,21 +33,55 @@
 MAX_INT = np.iinfo(np.int32).max
 
 
+def _generate_mask_from_indices(indices, mask_length, inverse=False):
```

Comment:
I'd be happy with `_indices_to_mask` for a utility function. I'm okay to see this as `indices_to_mask` in `utils` somewhere. I also don't really think `inverse` provides a lot, when I'd even prefer the implementation to just involve a `logical_not` or a `~` at the end.